sdk/core/azure-core/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 - Fixed `Azure::DateTime::Parse()` validation if the result is going to exceed `9999-12-31T23:59:59.9999999` due to time zone, leap second, or fractional digits rounding up adjustments.
 
+- Fixed issue [3224](https://github.com/Azure/azure-sdk-for-cpp/issues/3224) on MacOS crashing application when azure logs are turned on.
```

Comment:
The wording of this changelog is a bit awkward to parse and read. Could we create the link to the issue like we do in Embedded C, where the issue is in the beginning?
https://github.com/Azure/azure-sdk-for-c/blob/main/CHANGELOG.md#bugs-fixed

And then we could write it as follows:
[[3224]] Fixed intermittent crash on MacOS when logging is turned on.